app/scripts/constants.js

```diff
@@ -109,6 +109,11 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     namespace: "openshift"
   },
 
+
+  // Set to number of minutes after which a modal with countdown should appear.
+  // When set to anything else then positive integer the inactivity timeout won't be enable. 
+  INACTIVITY_TIMEOUT_PERIOD: 1,
```

Comment:
Think this accidentally got checked in enabled

Reply:
ohh... was retesting and accidentally forgot to disable it :-/